src/Nova/Booking.php

```diff
@@ -46,18 +46,20 @@ public static function indexQuery(NovaRequest $request, $query)
             'Reservationist',
         ])) {
             return $query
-                ->select('bookings.*')
-                ->leftJoin('slots as slot', 'slot.id', '=', 'bookings.slot_id')
-                ->leftJoin('rooms as room', 'room.id', '=', 'slot.room_id');
+                ->select('bookings.*');
```

Comment:
Hi @drewroberts

1 - Slot and rooms are not columns in the table `bookings`, there is only `slot_number` ... I prefer to comment instead of deleting.

2 - is necessary to have the create `booking` in the nova resource?, there are several columns marked as `exceptOnForms` .

Reply:
No, we do not want to create a Booking in Nova. There is a Nova Tool for that:

https://github.com/tipoff/staff-bookings